src/libsync/owncloudtheme.cpp

```diff
@@ -53,7 +53,7 @@ QColor ownCloudTheme::wizardHeaderTitleColor() const
 
 QPixmap ownCloudTheme::wizardHeaderLogo() const
 {
-    return QPixmap(hidpiFileName(":/client/theme/colored/wizard_logo.png"));
+    return QIcon(QStringLiteral(":/client/theme/colored/wizard_logo.png")).pixmap({132, 63});
```

Comment:
As discussed on RocketChat:

We need to change ownBrander for this anyway and so while at it, we want to make this function return a QIcon directly, so the resolution can be hardcoded wherever it's needed where the pixmap is actually consumed.